liquidation-monitor/agent-config.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Rename this file `bot-config.json`

Reply:
Updated in commit e08bc045b2f90f2087c0808b37ef836152ae46b4